website/templates/util/render_node.mako

```diff
@@ -1,10 +1,32 @@
 <%namespace name="contributor_list" file="./contributor_list.mako" />
 ## TODO: Rename summary to node
-<%def name="render_node(summary, show_path)">
+<%def name="render_node(index, summary, show_path)">
 ## TODO: Don't rely on ID
+<style>
```

Comment:
Belongs in style.css (ideally in osf-style, but that can be delayed). Add a comment to the style section where this is added that states it should be moved to osf-style.